buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -98,25 +99,15 @@ class VagrantTestPlugin implements Plugin<Project> {
     }
 
     private static void createBatsConfiguration(Project project) {
+        /* Setup the seed so it is available to bats tests if they want it and
+         * to the code below that sets up dependencies. */
+        RandomizedTestingPlugin.setupSeed(project)
```

Comment:
I don't think this should be necessary. This plugin should not be loaded unless we initialized the global build info, in which case we have already printed the seed. You should be able to just use the seed set on the root project.